app/components/mandaat/burgemeester-selector.hbs

```diff
@@ -0,0 +1,29 @@
+{{#if this.hasMultipleBurgemeesters}}
+  <div
+    class="au-o-box no-pagination au-u-padding-bottom-none au-u-margin-bottom-none"
```

Comment:
This no-pagination shouldn't be here, this is css for tables, don't think this has any effect here.